src/read_utils.rs

```diff
@@ -142,24 +142,26 @@ impl<'a, P: AsRef<Path>> FileHandler for &'a mut PathFileHandler<P> {
 /// Unlike ZipArchive, it gives access to a file by its name not regarding its path in the ZipArchive
 /// It thus cannot be correct if there are 2 files with the same name in the archive,
 /// but for transport data if will make it possible to handle a zip with a sub directory
-pub(crate) struct ZipHandler {
-    archive: zip::ZipArchive<File>,
+pub(crate) struct ZipHandler<R: std::io::Seek + std::io::Read> {
```

Comment:
I'm never sure which strategy to adopt but, do you need the bound here? I believe the bounds are only needed on the `impl` of `fn new()` and I know some `std` implementations put the bounds only were they're strictly needed. Any opinion on such methods?

Reply:
The bound is needed in `ZipArchive` no?

Reply:
Oops, you're right, forget about it :smile: